service_info/templates/cms/base.html

```diff
@@ -4,6 +4,11 @@
 <html>
   <head>
     <meta name="viewport" content="initial-scale=1.0, user-scalable=no" />
+    <meta property="og:url" content="{{ request.get_full_path }}" />
+    <meta property="og:type" content="website" />
+    <meta property="og:title" content="{% page_attribute 'page_title' %}" />
+    <meta property="og:site_name" content="Service Info" />
+    <meta property="og:locale" content="{% service_language_code 'Facebook' %}" />
```

Comment:
Can this new stuff get generated only "{% if request.current_page.ratingextension.include_rating %}"?  page generation otherwise might be a little cheaper